app/components/how-do-you-feel.jsx

```diff
@@ -0,0 +1,95 @@
+//https://github.com/EnCiv/civil-pursuit/issues/62
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+function Howdoyoufeel(props) {
+    const {
+        className = '',
+        onDone = () => { },
+        title = '',
+        disabled = false,
+        disableOnClick = false,
+        children,
+        ...otherProps
+    } = props
+    const [isDisabled, setIsDisabled] = useState(disabled)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.buttonBase, className)
+
+    return (
+        <div>
+            <p style={{ fontSize: "35px", fontWeight: "normal" }}>How do you feel about it</p>
+            <span>
+                <input className={combinedClassName}
+                    title={title} disabled={isDisabled} type="Submit" defaultValue="Awesome!"
+                    onClick={() => { onDone() }}
+                    {...otherProps} /> &nbsp;
+                <input className={combinedClassName}
+                    title={title} disabled={isDisabled} type="Submit" defaultValue="Just Okay"
+                    onClick={() => { onDone() }}
+                    {...otherProps} /> &nbsp;
+                <input className={combinedClassName}
+                    title={title} disabled={isDisabled} type="Submit" defaultValue="Unsatisfied"
+                    onClick={() => { onDone() }}
+                    {...otherProps} />
+            </span>
+        </div>
+    )
+}
+
+function HowdoyoufeelButton(props) {
+    const { className, ...otherProps } = props
+    const classes = buttonStyles()
+    return <Howdoyoufeel {...otherProps} className={cx(classes.Howdoyoufeel, className)} />
```

Comment:
Can't you just copy the code for Howdoyoufeel here and eliminate this function call.  It seems unusual.  Or, if there is a reason add a comment about why.  One thing I learned later in programming, is that the comments about why you did something are the most valuable over time.